<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoGFilter</class>
 <widget class="qMRMLWidget" name="DoGFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="main_grid">
   <item row="0" column="0">
    <widget class="QGroupBox" name="smooth_group">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Generate smoothed images</string>
     </property>
     <layout class="QGridLayout" name="smooth_grid">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="fwhm_input">
        <property name="maximum">
         <double>12.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>FWHM in mm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="btn_create_smooth_image">
        <property name="text">
         <string>Calculate smoothed image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="dog_group">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Difference of Gaussian combination settings</string>
     </property>
     <layout class="QGridLayout" name="dog_grid">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Second</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="first_image_cb"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="second_image_cb"/>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="btn_calculate_dog">
        <property name="text">
         <string>Calculate DoG image (first - second)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="result_group">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QListWidget" name="smooth_image_list">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>185</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="btn_view_image">
        <property name="text">
         <string>View selected image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="dog_image_list">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>185</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Export image as...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_remove_selected">
        <property name="text">
         <string>Remove selected</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btn_export_image">
        <property name="text">
         <string>Export selected image</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Smoothed images</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>DoG images</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_remove_all">
        <property name="text">
         <string>Remove all</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="qMRMLNodeComboBox" name="export_node">
        <property name="nodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="hideChildNodeTypes">
         <stringlist notr="true"/>
        </property>
        <property name="interactionNodeSingletonTag">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
